tests/test_testing.py

```diff
@@ -192,3 +197,40 @@ def on_post(self, req, resp):
     result = testing.simulate_post(
         app, '/', json={}, headers=headers, content_type=falcon.MEDIA_HTML)
     assert result.text == falcon.MEDIA_JSON
+
+
+@pytest.mark.parametrize('cookies', [
+    {'foo': 'bar', 'baz': 'foo'},
+    CustomCookies()
+])
+def test_create_environ_cookies(cookies):
+    environ = testing.create_environ(cookies=cookies)
+
+    assert environ['HTTP_COOKIE'] in ('foo=bar; baz=foo', 'baz=foo; foo=bar')
+
+
+def test_create_environ_cookies_options_method():
+    environ = testing.create_environ(method='OPTIONS', cookies={'foo': 'bar'})
+
+    assert 'HTTP_COOKIE' not in environ
+
+
+def test_cookies_jar():
+    class Foo:
+        def on_get(self, req, resp):
+            resp.set_cookie('has_permission', 'true')
```

Comment:
This is the only test for a single cookie (vs. multiple). I think it would be helpful for posterity to mention that here in a comment so that this particular input variation does not get lost if/when this test is ever modified. Alternatively, a separate test could be added that explicitly tests setting a single cookie in the request.

Reply:
I prefer to add a comment to the test, to avoid duplication of code(actually what will be).